training/general/src/environments/acpt-pytorch-1.11-cuda11.3/context/requirements.txt

```diff
@@ -7,9 +6,8 @@ azure-ml-component=={{latest-pypi-version}}
 azureml-mlflow=={{latest-pypi-version}}
 azureml-contrib-services=={{latest-pypi-version}}
 azureml-contrib-services=={{latest-pypi-version}}
+azureml-automl-common-tools=={{latest-pypi-version}}
 torch-tb-profiler~=0.4.0
```

Comment:
should we add torch-tb-profiler to common tools package?

Reply:
yeah I'll add it.